src/containers/BuySellCrypto/PaymentMethod/ManageWyreAccount/ManageWyrePersonalDetails.js

```diff
@@ -30,9 +31,9 @@ class ManageWyrePersonalDetails extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      name: this.props.individualLegalName.value,
-      dateOfBirth: this.props.individualDateOfBirth.value,
-      socialSecurityNumber: this.props.individualSsn.value,
+      name: typeof this.props.individualLegalName === 'undefined'? "" : this.props.individualLegalName.value,
```

Comment:
```this.props.individualLegalName == null ? "" : this.props.individualLegalName.value``` is better in this case because 

a) this.props.individualLegalName might be null, which would cause this to crash

b) no need to use typeof. It works because == in JS ignores the type difference between null and undefined.

Reply:
This is the checking for null comment referenced by other comments